blocks/app-download/app-download.css

```diff
@@ -58,6 +62,15 @@
   display: block;
 }
 
+.block.app-download .store a img {
+  width: 183px;
```

Comment:
Do we need both dimensions to be specified? Specifying both of them could potentially distort the image if that does not match image's aspect ratio.

Reply:
not need, only width required.